pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -534,8 +645,23 @@ def parse_args(args=None):
     ap.add_argument('--allow-backup', dest='allow_backup', default='true',
                     help="if set to 'false', then android won't backup the application.")
 
+    # Put together arguments, and add those from .p4a config file:
     if args is None:
         args = sys.argv[1:]
+
+    def _read_configuration():
+        if not exists(".p4a"):
+            return
+        print("Reading .p4a configuration")
+        with open(".p4a") as fd:
+            lines = fd.readlines()
+        lines = [shlex.split(line)
+                 for line in lines if not line.startswith("#")]
+        for line in lines:
+            for arg in line:
+                args.append(arg)
+    _read_configuration()
```

Comment:
What is this function for?
It seems to update `args`, but then it's being overridden again next lines :thinking:

Reply:
`args` before that overriding line is a string array, and afterwards is an ArgumentParser args object with the resulting values from parsing it. This approach is again historical - I suppose it could be renamed for clarity, I'll try to keep that in mind for the refactor later.

That parsing function itself is also historical from the webview and allows specifying additional argument strings that get injected into the parser, which I found quite neat - so I kept it for all bootstraps, since I saw no good reason to limit it. I could also remove it of course, but it does seem like something useful